sourcetracker/_sourcetracker.py

```diff
@@ -434,8 +434,25 @@ def calculate_cp_slice(self, xi, m_xiV, m_V, n_vnoti):
         return self.joint_probability
 
 
-def gibbs_sampler(sink, cp, restarts, draws_per_restart, burnin, delay):
-    """Run Gibbs Sampler to estimate feature contributions from a sink sample.
+def gibbs_sampler(sink, cp, restarts, draws_per_restart, burnin, delay,
+                  autoc=None):
+    '''Run Gibbs Sampler to estimate feature contributions from a sink sample.
+
+    Extended Summary
+    ----------------
+    This function can run in two modes. In the first mode, the Markov chain is
+    grown for a determined number of steps and values are taken at fixed
+    intervals. In the second mode, the values the chain produces are monitored
+    at each step, and if a convergence criterion is met, the function will
+    terminate rather than continuing to draw. This difference is reflected in
+    the following changes to parameter definition.
+
+    If `autoc` is not `None`, the meaning of the `draws_per_restart` parameter
+    changes. If `autoc` is `None`, `draws_per_restart` draws will be taken. If
+    `True`, this function will check for convergence in the number of counts in
```

Comment:
This description is a bit confusing. It reads like `autoc` takes either `None` or `True`.

I would change this line to say: *a function is provided, it will be applied to check...*